nearcore/src/metrics.rs

```diff
@@ -0,0 +1,28 @@
+use near_metrics::{try_create_histogram_vec, HistogramVec};
+use once_cell::sync::Lazy;
+
+pub static APPLY_CHUNK_DELAY: Lazy<HistogramVec> = Lazy::new(|| {
+    try_create_histogram_vec(
+        "near_apply_chunk_delay",
+        "Time process a chunk. Tgas used by the chunk is a metric label.",
+        &["tgas_ceiling"],
+        Some(prometheus::linear_buckets(0.0, 0.05, 50).unwrap()),
+    )
+    .unwrap()
+});
+
+pub static SECS_PER_THOUSAND_TGAS: Lazy<HistogramVec> = Lazy::new(|| {
+    try_create_histogram_vec(
+        "near_execution_seconds_per_petagas_ratio",
+        "Execution time per teragas. Ignore label 'label'.",
```

Comment:
The description here also needs an update.

Reply:
Done